avaframe/com1DFA/deriveParameterSet.py

```diff
@@ -744,42 +747,47 @@ def appendShpThickness(cfg):
     return cfg
 
 
-def checkDEM(cfgSim, demFile, onlySearch=False):
-    """ check if cell size of DEM in Inputs/ is same as desired meshCellSize
-        if not - check for remeshed DEM or remesh the DEM
+def checkRasterMeshSize(cfgSim, rasterFile, typeIndicator='DEM', onlySearch=False):
+    """ check if cell size of raster in Inputs/ is same as desired meshCellSize
+        if not - check for remeshed raster or remesh the raster
```

Comment:
suggestion: add info on onlySearch here in docstring description of function already